custom-recipes/pi-system-retrieve-list/recipe.json

```diff
@@ -112,6 +112,12 @@
             "type": "COLUMN",
             "columnRole": "input_dataset"
         },
+        {
+            "name": "do_duplicate_input_row",
+            "label": " ",
+            "description": " Copy input parameters in the output dataset",
+            "type": "BOOLEAN"
+        },
```

Comment:
Suggest renaming this to:
"Copy other input columns to the output dataset"

I'm not sure if it should just be a checkbox underneath - it should probably be it's own option with a label like
""Copy other input columns"

Reply:
donein [f8d9384](https://github.com/dataiku/dss-plugin-osisoft/pull/62/commits/f8d9384825d14ffd70692729185256e42414684d)